app/controllers/application_controller.rb

```diff
@@ -1,5 +1,24 @@
 class ApplicationController < ActionController::Base
+  # protect_from_forgery with: :exception
+  include SessionsHelper
+  include RestaurantsHelper
```

Comment:
Mấy cái helper này em dùng cho toàn bộ controller à mà phải include như thế này?

Reply:
k ạ @@